code/modules/mob/living/living.dm

```diff
@@ -515,6 +515,10 @@
 	if(loc && (istype(loc, /obj/structure/morgue) || istype(loc, /obj/structure/crematorium)))
 		resist_tray(L)
 
+	// The doctor left the odysseus unlocked again
+	if(src.loc && (istype(loc, /obj/item/mecha_parts/mecha_equipment/tool/sleeper)))
+		resist_mecha()
```

Comment:
This would probably be better if the above and wherever the locker resist one is were unified, see https://github.com/tgstation/-tg-station/blob/master/code/modules/mob/living/living.dm#L664-L668